### PR TITLE
Make the DSP work everywhere

### PR DESCRIPTION
**Pull request by geohot**
Doesn't work yet...but it should

### CODE REVIEW

File: launch_chffrplus.sh [hunk 1]
```diff
@@ -11,6 +11,11 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 function tici_init {
   sudo su -c 'echo "performance" > /sys/class/devfreq/soc:qcom,memlat-cpu0/governor'
   sudo su -c 'echo "performance" > /sys/class/devfreq/soc:qcom,memlat-cpu4/governor'
+
+  # init the DSP
+  sudo systemctl start adsprpcd
+  sudo systemctl start cdsp
+  sudo chmod 666 /dev/adsprpc-smd
```
Comment (adeebshihadeh) [inline launch_chffrplus.sh:18]: this should be done in a boot script

Comment (pd0wm) [reply launch_chffrplus.sh:18]: Shouldn’t it just be a udev rule?

File: common/realtime.py [hunk 2]
```diff
@@ -4,16 +4,21 @@
...
 
 
 # time step for each process
 DT_CTRL = 0.01  # controlsd
 DT_MDL = 0.05  # model
-DT_DMON = 0.1  # driver monitoring
 DT_TRML = 0.5  # thermald and manager
 
+# driver monitoring
+if TICI:
```
Comment (adeebshihadeh) [inline common/realtime.py:17]: we need to handle this in service_list at some point too. our comm issue check is wrong for frontFrame/driverState/dMonitoringState for example

Comment (pd0wm) [reply common/realtime.py:17]: Agree! Ideally these values should also be pulled from the service list.

**Pull request merged**